src/modules/groupTxsByDate.js

```diff
@@ -10,10 +10,10 @@ export const groupTxsByDate = (txs = []) => {
     if (date !== txDate) {
       date = txDate;
       dateIndex = query.length;
-      query.push({ timestamp: tx.timestamp, txs: [] });
+      query.push({ timestamp: tx.timestamp, data: [] });
```

Comment:
why use a different name?

Reply:
The SectionList component needs items to be called data